miniupnpd/netfilter_nft/nftnlrdr_misc.c

```diff
@@ -366,44 +366,47 @@ parse_rule_payload(struct nftnl_expr *e, rule_t *r)
 			*regptr = RULE_REG_IP_DEST_ADDR;
 			return;
 		} else if (offset == offsetof(struct iphdr, saddr) &&
-			   len == sizeof(in_addr_t)) {
+			       len == sizeof(in_addr_t)) {
```

Comment:
why indentation changes ?

Reply:
dunno, might have been my IDE 'helping'. Not intentional.

Reply:
Do you want me to resubmit?

Reply:
How do you have your tabs/spaces set up?

Reply:
Do you have a particular code sytle template you prefer? I'll configure my IDE to comply. Certainly don't want to create unnecessary work.

Reply:
just follow the style in the file.
To avoid commiting unnecessary changes, use `git add -pu` to only add relevant changes